package.json

```diff
@@ -45,59 +46,66 @@
     }
   },
   "dependencies": {
-    "@map-colonies/error-express-handler": "^1.2.0",
-    "@map-colonies/eslint-config": "^2.2.0",
-    "@map-colonies/mc-logger": "^2.0.2",
+    "@godaddy/terminus": "4.9.0",
```

Comment:
use the package https://github.com/MapColonies/error-types instead of common/errors.ts